packs/feats/mask-of-power.json

```diff
@@ -28,7 +28,51 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "option": "mask-of-power",
+                "placement": "spellcasting",
+                "suboptions": [
+                    {
+                        "label": "PF2E.SpecificRule.Spell.Fear.Label",
+                        "value": "fear"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Spell.PhantomPain.Label",
+                        "value": "phantom-pain"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Spell.SureStrike.Label",
+                        "value": "sure-strike"
+                    }
+                ],
+                "toggleable": true
+            },
```

Comment:
Hold off on doing these daily prep ones. I don't want to add a bunch of spell names to localization files unless necessary